airbyte-integrations/connectors/source-mssql/src/main/kotlin/io/airbyte/integrations/source/mssql/config_spec/MsSqlServerSourceConfigurationSpecification.kt

```diff
@@ -0,0 +1,122 @@
+    lateinit var database: String
+
+    @JsonProperty("schemas")
+    @JsonSchemaTitle("Schemas")
+    @JsonPropertyDescription("The list of schemas to sync from. Defaults to user. Case sensitive.")
+    // @DefaultSchemaDefault doesn't seem to work for array types...
+    @JsonSchemaInject(json = """{"order":3, "default":["dbo"], "minItems":0, "uniqueItems":true}""")
+    var schemas: Array<String>? = arrayOf("dbo")
+
+    @JsonProperty("username")
+    @JsonSchemaTitle("Username")
+    @JsonPropertyDescription("The username which is used to access the database.")
+    @JsonSchemaInject(json = """{"order":4}""")
+    lateinit var username: String
+
+    @JsonProperty("password")
+    @JsonSchemaTitle("Password")
+    @JsonPropertyDescription("The password associated with the username.")
+    @JsonSchemaInject(json = """{"order":5,"airbyte_secret":true}""")
+    lateinit var password: String
+
+    @JsonProperty("jdbc_url_params")
+    @JsonSchemaTitle("JDBC URL Params")
+    @JsonPropertyDescription(
+        "Additional properties to pass to the JDBC URL string when connecting to the database " +
+            "formatted as 'key=value' pairs separated by the symbol '&'. " +
+            "(example: key1=value1&key2=value2&key3=value3).",
+    )
+    @JsonSchemaInject(json = """{"order":6}""")
+    var jdbcUrlParams: String? = null
+
+    @JsonIgnore
+    @ConfigurationBuilder(configurationPrefix = "ssl_method")
+    var encryption = MicronautPropertiesFriendlyMsSqlServerEncryption()
+    @JsonIgnore var encryptionJson: MsSqlServerEncryptionConfiguration? = null
+    @JsonSetter("ssl_method")
+    fun setEncryptionValue(value: MsSqlServerEncryptionConfiguration) {
+        encryptionJson = value
+    }
+    @JsonGetter("ssl_method")
+    @JsonSchemaTitle("SSL Method")
+    @JsonPropertyDescription(
+        "The encryption method which is used when communicating with the database.",
+    )
+    @JsonSchemaInject(json = """{"order":7}""")
+    fun getEncryptionValue(): MsSqlServerEncryptionConfiguration? =
+        encryptionJson ?: encryption.asEncryption()
+
+    @JsonIgnore
+    @ConfigurationBuilder(configurationPrefix = "method")
+    var replicationMethod = MicronautPropertiesFriendlyMsSqlServerReplicationMethodConfiguration()
+    @JsonIgnore var replicationMethodJson: MsSqlServerReplicationMethodConfiguration? = null
+    @JsonSetter("replication_method")
+    fun setReplicationMethodValue(value: MsSqlServerReplicationMethodConfiguration) {
+        replicationMethodJson = value
+    }
+    @JsonGetter("replication_method")
+    @JsonSchemaTitle("Update Method")
+    @JsonPropertyDescription(
+        "Configures how data is extracted from the database.",
+    )
+    @JsonSchemaInject(json = """{"order":8, "default":"CDC", "display_type": "radio"}""")
+    fun getReplicationMethodValue(): MsSqlServerReplicationMethodConfiguration? =
+        replicationMethodJson ?: replicationMethod.asReplicationMethod()
+}
```

Comment:
I skimmed over this and the rest of the package but it seems legit. If your unit tests are happy then I'm happy.